src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacyContent.tsx

```diff
@@ -21,8 +20,8 @@ class ProjectDataPrivacyContent extends AsyncView<DataPrivacy> {
 
   getEndpoints() {
```

Comment:
This is how it's usually used in the codebase:
```suggestion
  getEndpoints(): [string, string][] {
```